admin/javascript/src/reducer-register.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * The register of Redux reducers.
+ * @private
+ */
+const register = {};
+
+/**
+ * The central register of Redux reducers for the CMS.
```

Comment:
for the Admin Section. Should it be for _all_ admin code, or just framework / cms module?